package.json

```diff
@@ -9,11 +9,14 @@
     "node": ">=14"
   },
   "dependencies": {
+    "@types/xml2js": "^0.4.9",
     "binascii": "^0.0.2",
     "crypto-random-string": "^3.3.1",
     "ejs": "^3.1.6",
+    "fast-plist": "^0.1.2",
     "ramda": "^0.27.1",
     "tslib": "^2.3.1",
+    "xml2js": "^0.4.23",
```

Comment:
If I'm not mistaken, those dependencies are only used in test and thus should go in the `devDependencies`.